wml_test_schedule

```diff
@@ -383,6 +383,9 @@
 9 four_cycle_recursion_branching
 9 four_cycle_recursion_by_id
 9 four_cycle_recursion_by_tagname
+9 abilitie_cycle_recursion_by_id
+9 filter_self_abilitie_cycle_recursion_by_id
```

Comment:
The singular of "abilities" is "ability".

Reply:
fixed

Reply:
Two of the filenames need fixing too.

Reply:
fixed